appformer-kogito-bridge/src/test/java/org/appformer/kogito/bridge/client/context/ChannelTest.java

```diff
@@ -30,6 +30,7 @@ public void withNameTest() {
         assertEquals(Channel.ONLINE, Channel.withName("ONLine"));
         assertEquals(Channel.VSCODE, Channel.withName("VSCode"));
         assertEquals(Channel.DESKTOP, Channel.withName("Desktop"));
+        assertEquals(Channel.EMBEDDED, Channel.withName("emBedDED"));
```

Comment:
Michael the weird cAsE here for the channel name is intentional?

Reply:
@ederign yEs! To be honest I thought I was copying existing tests with mixed case, assuming we were verifying the case does not affect resolution of the enum member.. however now you've asked the question I see only `ONLine` has unexpected mixed case and accept it's probably a typo! If you'd prefer me to change (or add more) just say the word.

Reply:
lEt'S meRGe tHIs